traffic_ops/app/lib/API/Tenant.pm

```diff
@@ -121,7 +115,12 @@ sub update {
 		}	
 	}	
 
-	if ( !defined( $params->{parentId}) && !$self->isRootTenant($id) ) {
+	my $tenant_utils = UI::TenantUtils->new($self);
+	my $tenants_data = $tenant_utils->create_tenants_data_from_db(undef);
+
```

Comment:
Indeed we agreed that a "root" tenant cannot become not root, and a non-root cannot become root. It also cannot be disabled.
Does it mean that a root tenant cannot be changed? I;m not sure but it indeed simolify things.
Will do